docs/stories/TF-1786-TeamMail-Swipe-in-Email-item-in-ThreadView .md

```diff
@@ -0,0 +1,97 @@
+# Summary
+
+* [Related EPIC](#related-epic)
+* [Definition](#definition)
+* [Screenshots](#screenshots)
+* [Misc](#misc)
+
+## Related EPIC
+
+* TODO: {link}
+
+## Acceptance Criteria: 
+```
+GIVEN I'm a Tmail user
+AND There are existed uread emails on my mailbox
+WHEN I swipe left to right on an email rows
+THEN I see the email row is swiped as blue background color belong with "read email icon"
+AND System mark an email as read
+
+GIVEN I'm a Tmail user
+AND There are existed read emails on my mailbox
+WHEN I swipe left to right on an email rows
+THEN I see the email row is swiped as blue background color belong with "unread email icon"
+AND System mark an email as unread
+
+GIVEN I'm a Tmail user
+AND There are existed emails on my normal folder which not archive folder
+WHEN I swipe right to left on an email rows
+THEN I see the email row is swiped as green background color belong with "archive email icon"
```

Comment:
We do not have an archive action.

Not sure archive is the right behaviour here.